config/migrations/2024/20241016165800-dedup-persons/20241016165804-clean-temp-graph.sparql

```diff
@@ -0,0 +1,18 @@
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+
+DELETE {
+  GRAPH ?g {
+    ?s owl:sameAs ?newS.
+  }
+}
+INSERT {
+  GRAPH <http://mu.semte.ch/graphs/public> {
```

Comment:
Is there a reason why this is written in public? Shouldn't this be in the correct graphs?

Reply:
i think it's good that:
- no one can modify this
- everyone can see that these are now the same